crates/re_sdk/src/global.rs

```diff
@@ -38,6 +38,38 @@ thread_local! {
     static LOCAL_BLUEPRINT_RECORDING: RefCell<Option<RecordingStream>> = RefCell::new(None);
 }
 
+/// Check whether a fork has happened since creating the recording streams.
+/// If so, then we forget our globals.
+pub fn cleanup_if_forked() {
```

Comment:
it's worth clarifying here if we are detecting the fork as the parent or as the child, here and elsewhere